ml-agents-envs/mlagents/envs/environment.py

```diff
@@ -355,11 +358,11 @@ def reset(
     @timed
     def step(
         self,
-        vector_action=None,
-        memory=None,
-        text_action=None,
-        value=None,
-        custom_action=None,
+        vector_action: Dict,
```

Comment:
Any chance you can be more specific on the values of these Dicts? I assume the types are strings?

Reply:
Done